src/containerapp/HISTORY.rst

```diff
@@ -8,6 +8,7 @@ upcoming
 * 'az containerapp containerapp create/up': `--registry-server` and `--source` use managed identity for image pull by default
 * 'az containerapp containerapp create': `--registry-server` use managed identity for image pull by default. `--no-wait` will not take effect with system registry identity.
 * 'az containerapp env java-component gateway-for-spring': Support create/update/show/delete Gateway for spring.
+* 'az containerapp env maintenance-config': Support Add, Update, Show, Delete
```

Comment:
Could you get ack from PMs with `maintenance-config` change since doc has different values

![image](https://github.com/user-attachments/assets/f21f6cd2-2c55-4752-bf42-e1a870f98f26)